docs/how-to/contribute/code.rst

```diff
@@ -1036,7 +1036,7 @@ For more information about Towncrier and fragment types see `News Fragments
 You can also see existing examples of news fragments in the ``changes``
 directory of the Toga repository. If this folder is empty, it's likely because
 Toga has recently published a new release; change note files are deleted and
-combined to update the :doc:`release notes </background/project/releases>` with
+combined to update the :doc:`release notes </about/releases>` with
```

Comment:
It's just occurred to me that this change has a side effect in the town crier configuration in `pyproject.toml` (as will the briefcase and rubicon analogs).